macros/parserDifferenceQuotient.pl

```diff
@@ -33,10 +33,14 @@ =head1 DESCRIPTION
 alphabetically is used to form the dx.  Otherwise, you can specify
 the variable used for dx as the second argument to
 DifferenceQuotient().  You could use a variable like h instead of
-dx if you prefer.
+dx if you prefer.  This is specified in the second argument.
+If you want to identify division by zero when a value other than
+zero is substituted in for dx (or h), then the third argument is a 
+number to be substituted into the variable named in the second 
+argument.  The third argument is optional and the default value
+0 is used when the third argument is omitted.
 
 =head1 USAGE
-
 	$df = DifferenceQuotient("2x+dx");
```

Comment:
Can you put back this blank line as well?  Sorry to be a pest.  You should be able to just add it to your branch and not make a new pull request.